client/my-sites/plugins/plugin-details-CTA/CTA-button.jsx

```diff
@@ -30,6 +30,7 @@ import { getDomainsBySiteId } from 'calypso/state/sites/domains/selectors';
 import { isSiteOnECommerceTrial } from 'calypso/state/sites/plans/selectors';
 import { isJetpackSite } from 'calypso/state/sites/selectors';
 import { getSelectedSite } from 'calypso/state/ui/selectors';
+import { getCategory } from '../categories/use-categories';
```

Comment:
1. Do you have any resources on understanding `JSX` files? I searched https://wordpress.org/search/jsx/, but did not find too many good results. 
2. Does `React` and `Redux` work out of the box in `JSX` files? I ask because I see traces of `dispatch` and `selectors` as well as hooks in `JSX` files.

Reply:
I think the [React Guide](https://reactjs.org/docs/introducing-jsx.html) is a better resource to understand how JSX is used with React.

> It is called JSX, and it is a syntax extension to JavaScript. We recommend using it with React to describe what the UI should look like. JSX may remind you of a template language, but it comes with the full power of JavaScript.

> JSX produces React “elements”. We will explore rendering them to the DOM in the [next section](https://reactjs.org/docs/rendering-elements.html). Below, you can find the basics of JSX necessary to get you started.

And as JSX files are basically Javascript files with some JSX markup on them, you can use everything you would use in a normal React file.

Reference: https://reactjs.org/docs/introducing-jsx.html